### ISSUE TITLE
[cluster actuator] Reconcile worker node security group

### ISSUE THREAD

**Issue by detiber** [kubernetes-sigs/cluster-api-provider-aws#51]
The cluster actuator should create and manage a security group for the control plane nodes if a user does not provide an existing security group.

Depends on: #38, #36

/kind feature
/priority important-soon

**Comment by chuckha** [comment]
closing in favor of #36
